### ISSUE TITLE
pip2 install fails

### ISSUE THREAD

**Issue by bbjay** [tiagocoutinho/multivisor#58]
I get the following error when installing multivisor 5.0.2
```
$ sudo pip2 install .
Processing /home/user/multivisor
Collecting flask (from multivisor==5.0.2)
  Using cached https://files.pythonhosted.org/packages/9b/93/628509b8d5dc749656a9641f4caf13540e2cdec85276964ff8f43bbb1d3b/Flask-1.1.1-py2.py3-none-any.whl
Collecting gevent (from multivisor==5.0.2)
  Using cached https://files.pythonhosted.org/packages/e6/e9/3a693414f706e12abe60554cd73c5ae8f848b182ae58018f93d86c9eb418/gevent-1.4.0-cp27-cp27mu-manylinux1_x86_64.whl
Requirement already satisfied: supervisor in /usr/lib/python2.7/dist-packages (from multivisor==5.0.2) (3.3.5)
Collecting zerorpc (from multivisor==5.0.2)
  Downloading https://files.pythonhosted.org/packages/73/ff/d61ef9f5d10e671421d1368e87d3525325483ebd7da262b1d3087443662b/zerorpc-0.6.3.tar.gz
Collecting louie (from multivisor==5.0.2)
  Using cached https://files.pythonhosted.org/packages/f2/f8/f9dfd97003f1c120dca1ed4dc9e3e16b74b583ce5bcb2d9b013142b6bee2/Louie-2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-2uxSEK/louie/setup.py", line 3, in <module>
        from louie import version
      File "louie/__init__.py", line 1, in <module>
        from . import dispatcher, error, plugin, robustapply, saferef, sender, signal, version
      File "louie/dispatcher.py", line 44
        f"Louie connects: {connects}\n"
        ^
    SyntaxError: invalid syntax
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-2uxSEK/louie/
```

Install with pip3 works, but then `$ multivisor` crashes with issues related to python3 incompatibility.

It looks like Louie v2.0 was released 3 months ago and dropped python 2.x support, so i needed to install an earlier version.
With `pip2 install Louie==1.1` it worked.

**Comment by tiagocoutinho** [comment]
Looks like louie produced a version 2.0 which is incompatible with python < 3.7 in September 2019.
However they don't restrict `python_requires` on they project.
I already created a PR in their project to prevent this trap.

**Comment by tiagocoutinho** [comment]
Should be fixed by #59. Thanks @bbjay!
